core/src/impl/Kokkos_ViewTracker.hpp

```diff
@@ -95,6 +137,10 @@ struct ViewTracker {
     if (view_traits::is_managed &&
         Kokkos::Impl::SharedAllocationRecord<void, void>::tracking_enabled()) {
       m_tracker.assign_direct(vt.m_track.m_tracker);
+#if defined(KOKKOS_ENABLE_VIEW_HOOKS)
+      if (Kokkos::Experimental::ViewHooks::is_set())
+        Kokkos::Experimental::ViewHooks::call(vt);
```

Comment:
Please don't do one-line `if` statements without `{}` (_especially_ inside of a macro)